eBay-brand-tracker/ebay_bt_venv/Lib/site-packages/automat/_runtimeproto.py

```
@@ -0,0 +1,62 @@
+"""
+Workaround for U{the lack of TypeForm
+<https://github.com/python/mypy/issues/9773>}.
+"""
+
+from __future__ import annotations
+
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)